### PR TITLE
approvedwithchat

### PR DESCRIPTION
**Pull request by yadavalliEmmanuel**

### CODE REVIEW

**Review by Bhaskar70 — APPROVED** [review]

File: src/app/components/user-page/user-page.component.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <div class="main-userDetails">
       <div class="Profile">
         <p class="userslabel">Profile Image</p>
-        <img src="{{ UserData.profileImageUrl }}" alt="" />
```
Comment (ResourceOne2011) [inline src/app/components/user-page/user-page.component.html:13]: why did u comment this img??

File: src/app/services/chat.service.ts [hunk 2]
```diff
@@ -21,10 +23,14 @@ export class ChatService {
...
   }
+  // chat Behavior
+  chatRequests(data: any) {
+    this.chatRequest.next(data);
+  }
   BE_SERVER = 'https://task-tracker-server-2njm.onrender.com';
   BE_LOCAL = 'http://192.168.10.30:1234';
   BE_LOCAL2 = 'http://192.168.29.109:1234';
-  BE_URL = this.BE_SERVER;
+  BE_URL = this.BE_LOCAL;
```
Comment (ResourceOne2011) [inline src/app/services/chat.service.ts:33]: change this to server url

File: src/app/components/user-page/user-page.component.html (hunk 1 above)
Comment (yadavalliEmmanuel) [reply src/app/components/user-page/user-page.component.html:13]: done

File: src/app/services/chat.service.ts (hunk 2 above)
Comment (yadavalliEmmanuel) [reply src/app/services/chat.service.ts]: done

**Review by ResourceOne2011 — APPROVED** [review]
OK, routing changed client to user

**Pull request merged**